content/docs/getting-started/hello-world.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Hello world"
+date: 2019-04-01T10:56:52+01:00
+menu:
+  docs:
+    parent: 'getting-started'
+weight: 101
+---
+
+Let's define an application that's capable of receiving a "hello {name}" message.
+
+```yaml
+asyncapi: '2.0.0'
+id: hello-world-app
+channels:
+  hello:
+    subscribe:
+      message:
+        payload:
+          type: string
+          pattern: '^hello .+$'
+```
+
+Let's get into the details of what we have here:
+
+<pre class="language-yaml line-numbers" data-line="1,2"><code>asyncapi: '2.0.0'
+id: hello-world-app
+channels:
+  hello:
+    subscribe:
+      message:
+        payload:
+          type: string
+          pattern: '^hello .+$'</code></pre>
+
+The first line tells us this is an AsyncAPI document and it's using version 2.0.0 of the specification. This line doesn't have to be the first one but it's a recommended practice.
+
+The second line is an identifier for our application. It is mandatory and must be unique so, when doing real stuff, don't use `hello-world-app` but instead something more "unique" like `urn:com:mycompany:hello-world-app`.
+
+<pre class="language-yaml line-numbers" data-line="3-9"><code>asyncapi: '2.0.0'
+id: hello-world-app
+channels:
+  hello:
+    subscribe:
+      message:
+        payload:
+          type: string
+          pattern: '^hello .+$'</code></pre>
+
+And here is the `channels` section. A channel is the medium where the messages flow through. In some systems this is known as `topic`, `event name` or `routing key`. Like in TV channels, you may receive different kind of information on each one.
```

Comment:
Suggestion for brevity: "The 'channels' section of the specification houses all of the mediums where messages flow through. For example, some systems use 'topic, 'event name' or 'routing key'. Different kinds of information flow through each channel."